tests/copyright_check/copyright_validator.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The library supports only Python3, I'm wondering if you could provide typing annotations for the defined functions